site/content/en/docs/Installation/_index.md

```diff
@@ -93,11 +80,20 @@ om-swaggerui-867d79b885-m9q6x       0/1     ContainerCreating   0          3m54s
 om-synchronizer-7f48f84dfd-j8swx    0/1     ContainerCreating   0          3m54s
 ```
 
-Open Match needs to be customized to your Matchmaker. This custom configuration is provided to the Open Match components via a ConfigMap (`om-configmap-override`). Thus, starting the core service pods waits on this config map being available.
+Open Match needs to be customized to your Matchmaker. This custom configuration is provided to the Open Match components via a ConfigMap (`om-configmap-override`). Thus, starting the core service pods waits on this config map to become available.
 
-If you are proceeding with the [Getting Started]({{< relref "../Getting Started/_index.md" >}}), or following a [Tutorial]({{< relref "../Tutorials/_index.md" >}}), these guides have steps to deploy their custom configuration and the Open Match is installed and ready to proceed.
+If you are proceeding with the [Getting Started]({{< relref "../Getting Started/_index.md" >}}), or following the [Tutorials]({{< relref "../Tutorials/_index.md" >}}), proceed with this guide to [install the default Evaluator]({{< ref "#install-the-default-evaluator" >}}) and configure Open Match to use it.
 
-If you are building a matchmaker outside of these guides, use the [Customization Guide]({{< relref "../Guides/custom.md" >}}) for steps to customize your Open Match installation.
+If you are building your own custom matchmaker and need to deploy a custom evaluator, then skip the next step and instead use the [Customization Guide]({{< relref "../Guides/custom.md" >}}) for steps to customize your Open Match installation.
+
+## Install the default evaluator
+
+Run the below command to install the default Evaluator in the open-match namespace and to configure Open Match to use the default installation.
```

Comment:
> ... and to configure Open Match to use the default installation.
It is not clear what the default installation refers to. Perhaps `and to configure Open Match to use it`?

Reply:
Done.